### PR TITLE
LPS-169355 Fetch default options values if the picklist value translation key does not yet exist

### PR DESCRIPTION
**Pull request by MirnaGama**
Bug found: [LPS-169355](https://issues.liferay.com/browse/LPS-169355)
Related to: [LPS-171482](https://issues.liferay.com/browse/LPS-171482)

### CODE REVIEW

**Comment by MirnaGama** [comment]
ci:test:sf

**Comment by MirnaGama** [comment]
ci:test:sf

**Comment by MirnaGama** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java [hunk 1]
```diff
@@ -172,14 +172,20 @@ protected List<Map<String, String>> getOptions(
...
 		if ((objectDefinition != null) &&
 			Validator.isNotNull(objectFieldName)) {
 
-			return _getOptionsFromObjectField(
+			options = _getOptionsFromObjectField(
 				objectDefinition.getObjectDefinitionId(), objectFieldName);
+
+			if (options.size() > 0) {
+				return options;
+			}
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java]: @MirnaGama can you check if we can use the `ListUtil.isNotEmpty` instead? `options` can be `null`

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by carolmariaabb** [comment]
ci:stop

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java (hunk 1 above)
Comment (MirnaGama) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java]: @carolmariaabb done !

**Comment by MirnaGama** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]
LGTM!

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**